### PR TITLE
Increase Melee damage by 30% across the board

### PR DESCRIPTION
**Pull request by yackemflam**
This'll give a better chance for those who uses melee weapons in a harmful manner.

### CODE REVIEW

**Comment by jughu** [comment]
Chaplain validhunt GOT BETTER!@!@$!@!#!w!@#@!

**Comment by Niknakflak** [comment]
Is this a meme

**Comment by Chocobro** [comment]
:-1:

**Comment by nullbear** [comment]
needs meme PR tag

**Comment by kevinz000** [comment]
:-1:

**Comment by Dorsisdwarf** [comment]
I dont think it's a meme, just a shitty idea.

We need a "Shitty idea" tag

**Comment by Jazaen** [comment]
Well, that is new.
Speaking of PR itself, it shits on sec, nukeops, revs and gangs (cargonia, gangtool weapons), while buffing wizard, changeling, blob and makes validhunt weapons stronger. No one has ever done that in just one PR!

**Comment by MMMiracles** [comment]
A meme to surpass metal gear.

**Comment by yackemflam** [comment]
@Jazaen How does it buff blob or changeling?
Also, any nuke operative can buy a dual esword and shit on anyone easily. It's just a MELEE weapon buff

**Comment by AdamElTablawy** [comment]
>unironically buffing every single weapon across the entire board for no reason without considering the balance connotations

i could add dildos and it'd be less retarded than this PR

**Comment by Jazaen** [comment]
Blob does brute, and changelings have their armblades.

**Comment by Shadowlight213** [comment]
@adamElTablawy
I think we have a dragon dildos sprite actually.

**Comment by Incomptinence** [comment]
Blood must flow.

Is good.

**Comment by Man-Shroom-** [comment]
I don't think it would be possible to consider the sweeping balance changes this PR adds outside of the actual game so I nominate that this be test-merged this saturday and sunday. We'll let everyone know in advance so they can avoid it or join in on the testing if they want.

**Comment by ChangelingRain** [comment]
Blobs don't use force(or do JUST brute) so this is actively a blob nerf

**Comment by yackemflam** [comment]
@ChangelingRain Welders/lasers still do more damage anyhow.

**Comment by Supermichael777** [comment]
>testmerge

fuck off

**Comment by MMMiracles** [comment]
test merging is actually a good thing because then you can show how shit it is in game

File: code/game/objects/items/weapons/melee/energy.dm [hunk 1]
```diff
@@ -23,10 +23,10 @@
 	name = "energy axe"
 	desc = "An energised battle axe."
 	icon_state = "axe0"
-	force = 40
-	force_on = 150
-	throwforce = 25
-	throwforce_on = 30
+	force = 52
+	force_on = 195
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/energy.dm]: literally a 1 hit kill

**Comment by Bawhoppen** [comment]
oh my god this is the worst pr of the year
congrats :clap: :clap: :clap: :clap:

File: code/game/objects/items/weapons/tools.dm [hunk 2]
```diff
@@ -284,7 +284,7 @@
 			return
 	//Welders left on now use up fuel, but lets not have them run out quite that fast
 		if(1)
-			force = 15
+			force = 20 //Warning, may fuck up the blob
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/tools.dm]: "how does this effect blob???"

dont think you can play dumb

File: code/game/objects/items/weapons/melee/energy.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/objects/items/weapons/melee/energy.dm]: it was basically a 1 hit already

**Comment by JJRcop** [comment]
@Jazaen Read the changes, this is not increasing all brute damage, it's increasing the damage values of melee weapons individually.

File: code/game/objects/items/weapons/twohanded.dm [hunk 3]
```diff
@@ -188,13 +188,13 @@
...
-	throwforce = 5
+	force = 4
+	throwforce = 7
 	throw_speed = 3
 	throw_range = 5
 	w_class = 2
-	force_unwielded = 3
-	force_wielded = 34
+	force_unwielded = 4
+	force_wielded = 44 // I am death, destroyer of worlds
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/twohanded.dm]: jesus christ

File: code/game/objects/items/weapons/melee/energy.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/energy.dm]: nah it was 2 hit

this is 5 damage from instant death, and is effectively an instant kill

**Comment by nullbear** [comment]
Should actually nerf force of all weapons, and get more robust instead.

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 4]
```diff
@@ -149,9 +149,9 @@
 	singular_name = "floor tile"
 	desc = "Those could work as a pretty decent throwing weapon."
 	icon_state = "tile"
-	force = 6
+	force = 12
```
Comment (KazeEspada) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: This isnt 30%

**Comment by ChangelingRain** [comment]
Here's the general opinion: No.

File: code/game/objects/items/stacks/tiles/tile_types.dm (hunk 4 above)
Comment (Iamgoofball) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: i wonder how many other "30%" stealth buffs are in here he didnt tell us about :^)

File: code/game/mecha/combat/marauder.dm [hunk 5]
```diff
@@ -54,7 +54,7 @@
 	health = 550
 	wreckage = /obj/structure/mecha_wreckage/seraph
 	internal_damage_threshold = 20
-	force = 55
+	force = 72
```
Comment (KazeEspada) [inline code/game/mecha/combat/marauder.dm]: Buffing mecha melee which is already their best attack

File: code/game/objects/items/weapons/tools.dm (hunk 2 above)
Comment (KazeEspada) [inline code/game/objects/items/weapons/tools.dm]: Its welders. It does double damage.

File: code/game/objects/items/weapons/twohanded.dm (hunk 3 above)
Comment (KazeEspada) [inline code/game/objects/items/weapons/twohanded.dm]: The weapon sucked offensively anyways

File: code/game/mecha/combat/marauder.dm (hunk 5 above)
Comment (Iamgoofball) [inline code/game/mecha/combat/marauder.dm]: that doesnt look like 30% either

Comment (yackemflam) [inline code/game/mecha/combat/marauder.dm]: 55 x 1.3 = 71.5

Comment (Iamgoofball) [inline code/game/mecha/combat/marauder.dm]: what do you have to say about floor tiles then nerd

Comment (yackemflam) [inline code/game/mecha/combat/marauder.dm]: Typo

Comment (Iamgoofball) [inline code/game/mecha/combat/marauder.dm]: yeah right

**Comment by PKPenguin321** [comment]
Knowledge-filled PR

File: code/game/mecha/combat/marauder.dm (hunk 5 above)
Comment (nullbear) [inline code/game/mecha/combat/marauder.dm]: lmao 2-hit-kill

Comment (Man-Shroom-) [inline code/game/mecha/combat/marauder.dm]: >goofball in charge of not sneaking shit in secretly

Comment (yackemflam) [inline code/game/mecha/combat/marauder.dm]: I can revert the mech damage if its that bad.

Comment (nullbear) [inline code/game/mecha/combat/marauder.dm]: it was already a 2-hit-kill.

But dun do this, it's closed. Do the generalized damage multiplier config thing

**Comment by MMMiracles** [comment]
when your pr is responsible for the birth of a new meme tab then you know you've achieved greatness

**Pull request closed without merging**